src/ceph-disk/ceph_disk/main.py

```diff
@@ -2377,13 +2379,14 @@ def set_or_create_partition(self):
                       self.args.lockbox)
             self.partition = self.create_partition()
 
-    def create_key(self):
+    def create_key(self, cluster):
```

Comment:
there is no need for requiring cluster here, you can rely on `self.args` since that will have the arguments passed in:

    self.args.cluster